cypress/e2e/eda/Decision-Environments/decision-env-crud.cy.ts

```diff
@@ -57,7 +57,7 @@ describe('EDA decision environment- Create, Edit, Delete', () => {
   it('can delete a decision environment from the details page', () => {
     cy.createEdaDecisionEnvironment().then((edaDE) => {
       cy.navigateTo(/^Decision environments$/);
-      cy.get('h1').should('contain', 'DecisionEnvironments');
+      cy.hasTitle(/^Decision environments$/);
       cy.get('button[aria-label="table view"]').click();
```

Comment:
This change removes an assertion. if you want to make this change, let's update the custom command to include a specific assertion

Reply:
updated